packages/server/service/index.js

```diff
@@ -35,4 +41,8 @@ const pgUrl = ({ username = "postgres", password = "postgres", host = "localhost
   return `postgresql://${username}:${enc(password)}@${host}:${port}/${database}${sslmode ? `?sslmode=${enc(sslmode)}` : ''}`
 }
 
+const parseMaybeInt = (str) => {
+  return str === undefined ? undefined : parseInt(str, 10)
```

Comment:
Handling empty env variables requires a little extra care, since we want to treat `undefined` and `''` identically, e.g. if the env is set as `DB_POOL_SIZE=`.  Something like this might do the trick:
```suggestion
  return str ? parseInt(str, 10) : undefined
```

Reply:
ah yup good call 👍